lighthouse-core/test/lib/i18n/i18n-test.js

```diff
@@ -98,6 +113,44 @@ describe('i18n', () => {
     });
   });
 
+  describe('#registerLocaleData', () => {
+    it('installs new locale strings', () => {
+      const localeData = {
+        'lighthouse-core/test/lib/i18n/i18n-test.js | testString': {
+          'message': 'en-XZ cuerda!'
+        }
+      };
+      i18n.registerLocaleData('en-XZ', localeData);
+
+      const UIStrings = {testString: 'en-US string!'};
+      const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+      const formattedStr = i18n.getFormatted(str_(UIStrings.testString), 'en-XZ');
+      expect(formattedStr).toEqual('en-XZ cuerda!');
+    });
+
+    it('overwrites existing locale strings', () => {
+      const filename = 'lighthouse-core/audits/is-on-https.js';
+      const UIStrings = require('../../../../' + filename).UIStrings;
```

Comment:
any reason this is dynamic? seems like a static string would be easier to refactor when file is moved, etc.